app/src/main/java/com/ezlevup/ganbyeong24/presentation/screens/care_request/CareRequestViewModel.kt

```
@@ -104,6 +150,123 @@ class CareRequestViewModel(
+                                .time
+
+                // 시작일이 오늘 이전인지 확인
+                if (startDate != null && startDate.before(today)) {
+                    errors["careStartDate"] = "시작일은 오늘 이후여야 합니다"
+                }
+
+                // 종료일이 시작일보다 이전인지 확인
+                if (startDate != null && endDate != null && endDate.before(startDate)) {
+                    errors["careEndDate"] = "종료일은 시작일 이후여야 합니다"
+                }
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)